hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/PressureAwareCompactionThroughputController.java

```diff
@@ -54,7 +54,7 @@
   public static final String HBASE_HSTORE_COMPACTION_MAX_THROUGHPUT_LOWER_BOUND =
       "hbase.hstore.compaction.throughput.lower.bound";
 
-  private static final long DEFAULT_HBASE_HSTORE_COMPACTION_MAX_THROUGHPUT_LOWER_BOUND =
```

Comment:
Please keep this private, this should be used by core code only. In Test, you can directly provide value: 50L * 1024 * 1024